### DISCUSSION TITLE
:MetalsShowCfr command not working

### DISCUSSION THREAD

**Discussion by caenrique** [scalameta/nvim-metals#357] [Q&A] [answered]
Hi! I'm sure I'm missing something or doing something wrong but I cannot make the command `:MetalsShowCfr` work.
Running it in a file inside a project with Scala version `3.1.0` with

    package package.name

    case class Person(name: String)
gives me the message: `[nvim-metals] Unsupported extension`
whereas `:MetalsShowJavap` and `:MetalsShowJavapVerbose` work just fine

**Comment by ckipp01** [comment]
Hey @caenrique, what version of Metals are you using? You can do a `:MetalsInfo` and you should see it right up on top.

**Comment by caenrique** [comment, marked as answer]
I'm not setting a specific version. The output of `:MetalsInfo`:
# metals 0.10.9
 Note:
   Supported Scala versions:
     Scala 3: 3.0.0, 3.0.1, 3.0.2, 3.1.0, 3.1.1-RC1
     Scala 2:
       2.11.12
       2.12.15, 2.12.14, 2.12.13, 2.12.12, 2.12.11, 2.12.8, 2.12.9, 2.12.10
       2.13.7, 2.13.4, 2.13.5, 2.13.6, 2.13.0, 2.13.1, 2.13.2, 2.13.3

## Current settings
```json
{
  excludedPackages = { "akka.actor.typed.javadsl", "com.github.swagger.akka.javadsl" },
  showImplicitArguments = true,
  showInferredType = true,
  superMethodLensesEnabled = true
}
```

**Reply by ckipp01** [reply]
Ah yea, that's the issue. You need to update. Doing a `:MetalsUpdate` should do the trick and get you `0.11.2`. Keep in mind that `nvim-metals` doesn't "auto-update" mainly because we don't want to pay the cost of checking all the time or having it hard coded somewhere. So whenever a new version is released you'll need to `:MetalsUpdate`.

**Reply by ckipp01** [reply]
A totally different way to manage it is to use coursier. If you use `cs` to install metals and regularly do a `cs update` you can also tell `nvim-metals` to use that Metals that is globally installed by setting `useGlobalExecutable`.

https://github.com/scalameta/nvim-metals/blob/3312490ef74ea149121a82fde578a13b1921cef9/doc/metals.txt#L440-L449

**Reply by caenrique** [reply]
Oh, I didn't notice that! Thanks :)
